.ruby-gemset

```diff
@@ -0,0 +1 @@
+hasbeen-in
```

Comment:
I've moved away from all the userspace shenanigans and settled [on the brightbox ppas](https://github.com/yakshed/vm/blob/4393e1557f7855b2fdcec81cd457b31d8630d06b/ansible/group_vars/all#L54) in my VM. (Thanks for @railsbros-dirk and @moonglum) for the hint)